.github/workflows/pycafe-dashboards-in-CI.yml

```
@@ -0,0 +1,47 @@
+name: PyCafe Dashboards in CI
```

```suggestion
name: PyCafe Dashboards
```

All these actions are CI 😀 I would rename the file to match also.